### PR TITLE
Feat/dogu agent cli

### PR DESCRIPTION
**Pull request by oneofthezombies**
# PR Template

## Checklist

- [ ] E2E finished.
- [ ] Local, Self Hosted and Development environment test finished.
- [ ] Local, E2E, Self Hosted, Development and Production env variable checked.
- [x] (Backend only) DB migration file creation checked.
- [x] **If this is a API deleted or changed**: Check side effects for API deletion or related code change.

# Changes

The "dogu-agent-core" module was seperated to allow interoperability of "dogu-agent (app version)" and "dogu-agent (cli version).
It doesn't work on linux yet.
Modified to register "external unit" according to "DOGU_DEVICE_PLATFORM_ENABLED".
Temporarily, ".env.device-server", ... and "features", ... were searched in "nm-space/projects/dost".
The file "projects/dogu-agent/src/commands/run.ts" is the core file.
There are many temporary processes in this file. Please review the design direction.

### CODE REVIEW

**Comment by oneofthezombies** [comment]
Reviews
* Move features data to dogu-agent-core.
* Create Env by env-generator to dogu-agent (cli version) workspace.

File: packages/typescript-private/dogu-agent-core/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Device Server
```
Comment (yowpark) [inline packages/typescript-private/dogu-agent-core/README.md:1]: Should title of readme "Dogu Agent Core?"

**Comment by yowpark** [comment]
LGTM! Thanks for your huge work~~ 👍 👏

**Comment by oneofthezombies** [comment]
Review applied
- Move features data to dogu-agent-core.
- Create Env by env-generator to dogu-agent (cli version) workspace.

**Pull request merged**